corehq/apps/locations/views.py

```diff
@@ -807,3 +809,35 @@ def loc_to_payload(loc):
             locs = locs.filter(name__icontains=query)
 
         return json_response(map(loc_to_payload, locs[:10]))
+
+
+class DowngradeLocationsView(BaseDomainView):
```

Comment:
This page takes the place of the location pages if a domain gets downgraded, but still has users assigned to locations.  All it is some explanatory text and a button explaining what the problem is.